src/less/section-notice/base/section-notice.less

```diff
@@ -45,19 +43,16 @@ span[role="region"].section-notice {
     justify-content: center;
 }
 
-.section-notice__main {
-    flex: 1;
-}
-
-.section-notice__title {
-    font-size: @font-size-14;
-    margin: 0 0 4px 0;
+.section-notice :is(h1, h2, h3, h4, h5, h6) {
+    font-size: initial;
+    font-weight: normal;
+    margin: 0;
 }
```

Comment:
Personally, I'm not sure I like this. I think we should not force headings to be a certain size and weight. 
If someone wants to put a heading in the notice, so be it, let them deal with it, don't make it harder for our users.

Reply:
Yes, let's hold off on any drastic changes to the headings.